packages/vite-extensions/vite-website-cache-paths.ts

```diff
@@ -0,0 +1,93 @@
+import { readdirSync, statSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+/**
+ * We want to cache all files from the website root directory, and the assets directory, except for the files listed below.
```

Comment:
Assets is under website root so we should be good to skip it from the comment:

```suggestion
 * We want to cache all files from the website root directory, except for the files listed below.
```